### DISCUSSION TITLE
Historical Office Headlines

### DISCUSSION THREAD

**Discussion by UnRealDinnerbone** [weather-gov/api#632] [Q&A]
I was wondering if there is a way though the API to receive historical headlines for an office. For example, if you go https://www.weather.gov/lot/events you will see two active headlines at the top of the page (at least at the time of writing this) and those two events are what the API responds with (https://api.weather.gov/offices/LOT/headlines). I was wondering if from the API I could get the old headlines that are shown farther down on the page

**Comment by sullynole** [comment]
The headlines exposed on https://api.weather.gov/offices/LOT/headlines are the current news headlines at the top of all https://www.weather.gov/lot office pages.  The content further down the page on https://www.weather.gov/lot/events is completely separate and may or may not have been a past headline.  Regardless, only current headlines are available through the api.

**Comment by UnRealDinnerbone** [comment]
Alright thanks
